### PR TITLE
updates to Pro 2.5

### PR DESCRIPTION
**Pull request by lingtangraster**
@hanhu7 Please review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,12 +106,24 @@ We provide deep learning model inference Python raster function (PRF) for a list
     [Sample emd file](examples/keras/mask_rcnn/mask_rcnn.emd) |
     [Sample PRF](python_raster_functions/Keras/MaskRCNN.py)**]
 
+* [Keras image classification](https://www.geeksforgeeks.org/python-image-classification-using-keras/) (For ArcGIS Pro 2.5 and Enterprise 10.8 and later)
+  
+    [**[Sample case](examples/keras/object_classification/README.md) | 
+    [Sample emd file](examples/keras/object_classification/model/HouseDamageClassifier_ProBuiltin.emd) |
+    [Sample PRF](python_raster_functions/Keras/KerasClassifier.py)**]
```
Comment (hanhu7) [inline README.md]: Is the python file missing?

File: README.md [hunk 2]
```diff
@@ -106,12 +106,24 @@ We provide deep learning model inference Python raster function (PRF) for a list
...
-    [Sample emd file](examples/pytorch/pytorch_fastai_ssd.emd) |
+    [**[Sample case](examples/pytorch/object_detection/README.md) | 
+    [Sample emd file](examples/pytorch/object_detection/pytorch_fastai_ssd.emd) |
     [Sample PRF](python_raster_functions/PyTorch/FastaiSSD.py)**]
 
+* [Object Classification implementation with PyTorch](https://developers.arcgis.com/python/sample-notebooks/building-damage-assessment-using-feature-classifier/) (For ArcGIS Pro 2.5 and Enterprise 10.8 and later)
+
+    [**[Sample case](examples/pytorch/object_classification/README.md) | 
+    [Sample emd file](examples/pytorch/object_classification/woolseyFire_600_50.emd) |
+    [Sample PRF](python_raster_functions/PyTorch/FeatureClassifier.py)**]
```
Comment (hanhu7) [inline README.md]: Is the python file missing?

File: examples/keras/object_classification/README.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# Keras Mask RCNN House Footprints Example in ArcGIS Pro
+Step 0. Download test deep learning model, input image, input features from the link (TBD).
+You can also use your own trained model and test image. 
+
+Step 1. Open "Detect Object Using Deep Learning"
```
Comment (hanhu7) [inline examples/keras/object_classification/README.md]: Should this be open "Classify Objects ... "?

Comment (lingtangraster) [reply examples/keras/object_classification/README.md]: done

File: README.md (hunk 2 above)
Comment (lingtangraster) [reply README.md]: done

**Pull request merged**